package/tests/testthat/_snaps/pkct01/markdown-snaps.md

```diff
@@ -8,27 +8,96 @@
-          2                               134       0        13.4     1.35       10.1            13.4               10.0            13.3     10.8      16.5  
-          3                               134       0        8.47     1.25       14.7            8.38               15.0            8.4      5.88      10.9  
-          4                               134       0        4.79     1.02       21.2            4.69               22.0            4.79      2.7      7.09  
-          8                               134       0       0.348     0.18       51.7           0.303               58.4           0.318     0.076     0.866 
-          12                              134       0       0.0224   0.0189      84.6           0.0156              111.6          0.017     0.002     0.083 
-        Day 2                                                                                                                                                
-          24                              134      134        0        0          NE              NE                 NE              0         0         0   
-      ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      Treatment Group                                                                                                                                               
+        Visit                                                                                                                                                       
+          Nominal Time from First Dose (hr)          Number                                                                                                         
+                                                       of                                                                                                           
+                                               n    LTRs/BLQs    Mean       SD     CV (%) Mean   Geometric Mean   CV % Geometric Mean   Median    Minimum    Maximum
+      ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      A: Drug X (N=134)                                                                                                                                             
+        Day 1                                                                                                                                                       
+          0                                   134      134       0        0            NE            0                    NE             0         0          0     
```

Comment:
@edelarua not sure that the 0 should display here for geometric mean?

Per GDSR:
![image](https://github.com/insightsengineering/tlg-catalog/assets/43623360/9b58a63c-6a9e-4b81-b28d-00fd610b9ff9)